ocr_service/core/config.py

```
@@ -0,0 +1,37 @@
+import os
+from typing import Dict, List
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)